docs/aws-sdk-for-javascript-with-minio.md

```diff
@@ -23,7 +23,7 @@ var s3  = new AWS.S3({
           accessKeyId: 'YOUR-ACCESSKEYID' ,
           secretAccessKey: 'YOUR-SECRETACCESSKEY' ,
           endpoint: 'http://127.0.0.1:9000' ,
-          s3ForcePathStyle: true, // needed with minio?
+          s3ForcePathStyle: true, // required for minio - changes requests from <bucket>.host to host/<bucket>
```

Comment:
this is really not necessary if you happen to use MINIO_DOMAIN to enable virtual style requests, perhaps we should mention that here.  But by default yes.